doc/migration/version-upgrade-guide.md

```diff
@@ -1175,15 +1175,10 @@ install:
 - The `--namespace` flag from `operator-sdk run --local`, `operator-sdk test --local`, and `operator-sdk cleanup` was
 deprecated and is replaced by `--watch-namespace` and `--operator-namespace`.
 
-    The `--operator-namespace` flag can be used to set the namespace where the operator will be deployed. It will set the
-    environment variable `OPERATOR_NAMESPACE`. If this value is not set, then it will be the namespace defined as in your
-    current kubeconfig context.
+> **NOTES:** 
+> - The `--operator-namespace` flag can be used to set the namespace where the operator will be deployed. It will set the environment variable `OPERATOR_NAMESPACE`. If this value is not set, then it will be the namespace defined as in your current kubeconfig context.
 
-    The `--watch-namespace` flag can be used to set the namespace(s) which the operator will watch for changes. It will set
-    the environment variable `WATCH_NAMESPACE`. Use an explicit empty string to watch all namespaces or a comma-separated
-    list of namespaces (e.g. "ns1,ns2") to watch multiple namespace when the operator is cluster-scoped. If using a list,
-    then it should contain the namespace where the operator is deployed since the default metrics implementation will
-    manage resources in the Operator's namespace. By default, `--watch-namespace` will be set to the operator namespace.
+> - The `--watch-namespace` flag can be used to set the namespace(s) which the operator will watch for changes. It will set the environment variable `WATCH_NAMESPACE`. Use an explicit empty string to watch all namespaces or a comma-separated list of namespaces (e.g. "ns1,ns2") to watch multiple namespace when the operator is cluster-scoped. If using a list, then it should contain the namespace where the operator is deployed since the default metrics implementation will manage resources in the Operator's namespace. By default, `--watch-namespace` will be set to the operator namespace.
```

Comment:
This change makes the rendered markdown look strange. What's wrong with the existing formatting?